.github/workflows/test.yml

```diff
@@ -24,10 +24,12 @@ jobs:
       sessions: ${{ steps.set-matrix.outputs.sessions }}
     steps:
       - uses: actions/checkout@v4
-      - run: pip install poetry nox nox-poetry
+      - run: python3 -m venv .venv
```

Comment:
can we use uv instead? :D

Reply:
Sure! I didn’t change it earlier because I wanted to minimize changes, but I will make this changes on this PR.

Reply:
Updated!